### PR TITLE
[DTM] - User experience improved for complex property ordering

### PR DESCRIPTION
**Pull request by Vagdevi1998**
Improved UI for complex property ordering

### CODE REVIEW

File: content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-953
+version: 1020.1.4
+---
+
+Users can arrange the properties in the required order by dragging and dropping them.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md:18]: Please provide a bit more context. Where exactly?

Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md]: Modified

File: content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-Tl88RYb4A
+    label: Digital Twin Manager
+build_artifact:
+  - value: tc-wYIY0MBDO
+    label: dtm
+ticket: CTM-953
+version: 1020.1.4
+---
+
+Users can now order the complex property keys by dragging and dropping them using the drag icon which appears on hover.
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md:18]:
```suggestion
Users can now order the complex property keys by dragging and dropping them using the drag icon which appears on hovering over the row.
```

**Review by BeateRixen — APPROVED** [review]

File: content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: UI improvement for ordering of complex properties
+product_area: Application enablement & solutions
+change_type:
+  - value: change-2c7RdTdXo4
+    label: Fix
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md:7]: For this to be visible in CD change log, please change this to Improvement

**Review by ghost — CHANGES_REQUESTED** [review]

File: content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md (hunk 3 above)
Comment (Vagdevi1998) [reply content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md]: Modified

File: content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+---
+date:
+title: UI improvement for ordering of complex properties
+product_area: Application enablement & solutions
+change_type:
+  - value: change-2c7RdTdXo4
```
Comment (lra-sag) [inline content/change-logs/application-enablement/dtm-1020-1-4-ui-improvement-for-ordering-of-complex-properties.md:6]: Please check if this is the right identifier for Improvement change type

**Review by ghost — APPROVED** [review]

**Pull request merged**